app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -416,9 +41,25 @@ const ListingDetail = () => {
         </div>
         {listing && (
           <article className="flex flex-wrap relative max-w-5xl m-auto w-full">
-            {getImage()}
-            {getAMISection()}
-            {getBodyAndSidebar()}
+            <ListingDetailsImageCard listing={listing} />
+            <ListingDetailsPricingTable listing={listing} />
+            <ListingDetails>
+              <ListingDetailsEligibility listing={listing} />
+              <ListingDetailsFeatures listing={listing} />
+              <ListingDetailsNeighborhood />
+              <ListingDetailsAdditionalInformation listing={listing} />
+              <ListingDetailsAside listing={listing} />
```

Comment:
The names of these components ended up rather long. Although the directory will tell give us context for where the component is used, I was torn between this or naming them shorter e.g. `Features.tsx` and us being confused later on when there could be multiple `Features.tsx`, `Neighborhood.tsx` or `Eligibility.tsx`, etc components

Reply:
I'm fine with verbose names for now :)